backend/tests/unit/agents/test_kubernetes_agent.py

```
@@ -63,21 +64,23 @@ def test_initialization_with_required_dependencies(self, mock_llm_client, mock_m
         assert agent.mcp_registry is mock_mcp_registry
-        assert agent.progress_callback is None
         assert agent._iteration_count == 0
         assert agent._configured_servers is None
+        # Verify default iteration strategy
+        from tarsy.agents.constants import IterationStrategy
+        assert agent.iteration_strategy == IterationStrategy.REACT
     
-    def test_initialization_with_progress_callback(self, mock_llm_client, mock_mcp_client, mock_mcp_registry):
-        """Test KubernetesAgent initialization with progress callback."""
-        mock_callback = Mock()
+    def test_initialization_with_custom_iteration_strategy(self, mock_llm_client, mock_mcp_client, mock_mcp_registry):
```

Ruff (0.12.2)

Line too long (118 > 88)

(E501)